osu.Server.Spectator/Database/Models/database_queue_mode.cs

```diff
@@ -24,11 +24,11 @@ public static QueueMode ToQueueMode(this database_queue_mode mode)
                 case database_queue_mode.host_only:
                     return QueueMode.HostOnly;
 
-                case database_queue_mode.free_for_all:
-                    return QueueMode.FreeForAll;
+                case database_queue_mode.all_players:
+                    return QueueMode.AllPlayers;
 
-                case database_queue_mode.fair_rotate:
-                    return QueueMode.FairRotate;
+                case database_queue_mode.all_players_rr:
+                    return QueueMode.AllPlayersRR;
```

Comment:
hmm, i think this one should maybe be `AllPlayerRoundRobin`... not sure

Reply:
Renamed to the full name in all cases not too bad overall.